opennms-container/core/Dockerfile

```diff
@@ -85,6 +85,9 @@ RUN groupadd \
 # We would have 755 for opennms:root instead of 775 and prevents writing lock files in /usr/share/opennms
 COPY --chown=10001:0 --from=core-tarball /opt/usr-share /usr/share
 
+COPY plugins.sh /tmp/plugins.sh 
+RUN apt-get update && apt-get install -y python3-pip wget curl jq && pip3 install --upgrade cloudsmith-cli && chmod +x /tmp/plugins.sh  && cd /tmp && ./plugins.sh && rm ./plugins.sh && apt-get remove -y python3-pip wget curl jq && apt-get clean && rm -rf /var/cache/apt /var/lib/apt/lists/* /tmp/security.sources.list && rm -rf ~/.cache/pip
```

Comment:
it would be nice to break this up into multiple lines with `\` so diffs will be cleaner and it's easier to tell what's going on